### PR TITLE
Remove DaprInvokeHttpRoute from docs

### PR DESCRIPTION
**Pull request by rynowak**
This change removes mentions of the `daprInvokeHttpRoute` from docs as we're removing it from Radius.

Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.dev/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### Summary
🔄🗑️🧹

This pull request simplifies and updates the code snippets and documentation pages related to Dapr service invocation. It removes the use of routes, connections, and annotations, and replaces them with app IDs and environment variables. It also deletes the files and sections that are no longer needed or supported.

> _To invoke services with Dapr_
> _You don't need routes or a mapper_
> _Just use app IDs_
> _And some env vars_
> _And your code will be much snappier_

### Walkthrough
*  Simplify service invocation by using app IDs instead of routes ([link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-03c2b7651d4457db3cb0f4bc2078679158f72f4d2c62f0c2dce9686b28e5b89fL68-R71), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-a5d5b487913ce1322887a33aa00792edd7fc4d20a019352731e62ff532211ef7L39-R46))
*  Remove unused environment parameter from service invocation snippet ([link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-e3fea44d347b5c1399ccee25c6c8bdaea8d63d1f429d594b13924c08bdf1bdf6L3))
*  Remove provides property and daprRoute resource from backend port definitions in various snippets, as they are no longer needed to link to a route ([link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-e3fea44d347b5c1399ccee25c6c8bdaea8d63d1f429d594b13924c08bdf1bdf6L21), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-e3fea44d347b5c1399ccee25c6c8bdaea8d63d1f429d594b13924c08bdf1bdf6L27-L34), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-82ee426db5ea4a071df4465003c8fc8c3c49188baa0826d1c9c807b6638dda10L23), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-82ee426db5ea4a071df4465003c8fc8c3c49188baa0826d1c9c807b6638dda10L39), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-82ee426db5ea4a071df4465003c8fc8c3c49188baa0826d1c9c807b6638dda10L47-L55), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-79a518d1fc53fc5e56e65c936aad3b842f57ea6d3b8564f4b5041e053b5857a9L35), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-79a518d1fc53fc5e56e65c936aad3b842f57ea6d3b8564f4b5041e053b5857a9L43-L51), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-2921a3d53dbda5caa96df4bad5cb93934c113a4e707390a33b3e017596174524L40), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-2921a3d53dbda5caa96df4bad5cb93934c113a4e707390a33b3e017596174524L48-L56), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-fbb4c16461c94a47d8f6d9877e5b9baeb1f23f796524e74c85daf35a2f8fc0d6L35), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-fbb4c16461c94a47d8f6d9877e5b9baeb1f23f796524e74c85daf35a2f8fc0d6L43-L51), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-c0af3c43f1d2fc86932fc92f65ecbeb05948a8e4beedb6667cf309c2cf5cc34cL40), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-c0af3c43f1d2fc86932fc92f65ecbeb05948a8e4beedb6667cf309c2cf5cc34cL48-L56), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-17e6b7269c9d82adc4705d6d9dcba055850b1d522ad8085e2006e51fa6d751f7L40), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-17e6b7269c9d82adc4705d6d9dcba055850b1d522ad8085e2006e51fa6d751f7L48-L55))
*  Replace connections property with env property in frontend container definitions in various snippets, to use an environment variable to configure the app ID of the backend ([link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-e3fea44d347b5c1399ccee25c6c8bdaea8d63d1f429d594b13924c08bdf1bdf6L43-R35), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-fbb4c16461c94a47d8f6d9877e5b9baeb1f23f796524e74c85daf35a2f8fc0d6R48-R50), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-fbb4c16461c94a47d8f6d9877e5b9baeb1f23f796524e74c85daf35a2f8fc0d6L65-L69), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-c0af3c43f1d2fc86932fc92f65ecbeb05948a8e4beedb6667cf309c2cf5cc34cR53-R55), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-c0af3c43f1d2fc86932fc92f65ecbeb05948a8e4beedb6667cf309c2cf5cc34cL70-L74), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-17e6b7269c9d82adc4705d6d9dcba055850b1d522ad8085e2006e51fa6d751f7R55-R57), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-17e6b7269c9d82adc4705d6d9dcba055850b1d522ad8085e2006e51fa6d751f7L71-L75))
*  Update frontend container properties description in add UI quickstart page, to use an environment variable and daprSidecar extension instead of connection and annotation ([link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-63d449da98bb44453d9d2eaa90b2470670c7af227a75eca39297e9134e7383d1L17-R18))
*  Delete `invoke.bicep` snippet, as it is no longer needed to model service invocation ([link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-aca029e041e69252502bf14795b531b16a027f8b99dca22f03c830305c266c9c))
*  Remove appId and provides properties from dapr extension schema page, as they are no longer relevant for the extension ([link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-3d7cdc47166a47ffeb9e949bfd8a11c427163ce49089a5357e1e9a610d4d5bb4L25-R26))
*  Delete `dapr-http` schema page and snippet, as they are no longer supported by the Applications.Link resource provider ([link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-2ddb3fe1e8efb1282bc8d98c1930a03ba4f5f639ed48c00471e3db5084050941), [link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-06755f1445ade6da3385d99451b5fdef5f2c7cbe3063bbb8c2700727295c0ac4))
*  Remove daprInvokeHttpRoutes resource entries from resource policies page, as they are no longer supported by the Applications.Link resource provider ([link](https://github.com/project-radius/docs/pull/624/files?diff=unified&w=0#diff-cfaf17663b1c6fdd12427eb256879b86ecb1e0e4f80748334518244ef80a69a1L55-L56))

### CODE REVIEW

File: docs/content/author-apps/dapr-resources/snippets/service-invocation.bicep [hunk 1]
```diff
@@ -40,11 +30,9 @@ resource frontend 'Applications.Core/containers@2022-03-15-privatepreview' = {
...
-    }
-    connections: {
-      // Automatically inject environment variables
-      backend: {
-        source: daprRoute.id
+      env: {
+        // Configures the appID of the backend service.
+        CONNECTIONS_BACKEND_APPID: 'backend'
       }
     }
```
Comment (AaronCrawfis) [inline docs/content/author-apps/dapr-resources/snippets/service-invocation.bicep:37]: Frontend should also have a Dapr extension. I must have missed it last time around

File: docs/content/author-apps/dapr-resources/snippets/service-invocation.bicep [hunk 2]
```diff
@@ -40,11 +30,9 @@ resource frontend 'Applications.Core/containers@2022-03-15-privatepreview' = {
...
     container: {
       image: 'frontend:latest'
-    }
-    connections: {
-      // Automatically inject environment variables
-      backend: {
-        source: daprRoute.id
+      env: {
+        // Configures the appID of the backend service.
+        CONNECTIONS_BACKEND_APPID: 'backend'
```
Comment (AaronCrawfis) [inline docs/content/author-apps/dapr-resources/snippets/service-invocation.bicep]: CONNECTIONS or CONNECTION?

File: docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-ui/snippets/dapr-azure.bicep [hunk 3]
```diff
@@ -32,41 +32,29 @@ resource backend 'Applications.Core/containers@2022-03-15-privatepreview' = {
...
+        CONNECTIONS_BACKEND_APPID: 'backend'
+      }
       ports: {
         ui: {
           containerPort: 80
           provides: frontendRoute.id
         }
       }
     }
-    connections: {
```
Comment (AaronCrawfis) [inline docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-ui/snippets/dapr-azure.bicep:65]: Thoughts on opening an issue to add in functionality to continue to inject `CONNECTION_BACKEND_APPID` when there's a connection and the target has a Dapr sidecar extension?

Comment (rynowak) [reply docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-ui/snippets/dapr-azure.bicep:65]: Connection to what?

Comment (rynowak) [reply docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-ui/snippets/dapr-azure.bicep:65]: I think we should just design what we want the Dapr connection support to be.

Comment (AaronCrawfis) [reply docs/content/getting-started/quickstarts/quickstart-dapr/dapr-microservices-add-ui/snippets/dapr-azure.bicep:65]: Opened https://github.com/project-radius/radius/issues/5754 to track

File: docs/content/author-apps/dapr-resources/snippets/service-invocation.bicep (hunk 2 above)
Comment (rynowak) [reply docs/content/author-apps/dapr-resources/snippets/service-invocation.bicep]: Fixed (everywhere)

File: docs/content/author-apps/dapr-resources/snippets/service-invocation.bicep (hunk 1 above)
Comment (rynowak) [reply docs/content/author-apps/dapr-resources/snippets/service-invocation.bicep:37]: Will fix

**Comment by rynowak** [comment]
@AaronCrawfis updated

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**